### PR TITLE
rm the deprecate warning in v0.4

### PR DESCRIPTION
**Pull request by panlanfeng**
rm the deprecate warning in v0.4

### CODE REVIEW

**Comment by stevengj** [comment]
To avoid breaking 0.3, you need to (a) add `using Compat` and (b) wait for a new version of Compat to be tagged.

**Comment by stevengj** [comment]
Okay, I've tagged a new version of Compat.  You should also modify the `REQUIRE` file to require Compat 0.7.

**Comment by panlanfeng** [comment]
Just tested on both v0.3 and v0.4 and both work well.

File: deps/build.jl [hunk 1]
```diff
@@ -23,5 +23,5 @@ if !isfile(tagfile) || readchomp(tagfile) != cubvers
             run(`gcc -shared -o ../libcubature.so hcubature.o pcubature.o`)
         end
     end
-    run(`echo $cubvers` |> tagfile)
+    @compat run(pipeline(`echo $cubvers`, tagfile))
```
Comment (stevengj) [inline deps/build.jl]: You don't need `@compat` -- that macro is only for syntax changes.  `pipeline` is an ordinary function, and the Compat macro just exports a `pipeline(...)` replacement function in 0.3.

**Comment by panlanfeng** [comment]
I think this is ready to merge.

**Comment by stevengj** [comment]
Thanks!

**Pull request merged**